278_first_bad_version.py

```diff
@@ -0,0 +1,11 @@
+class Solution:
+    def firstBadVersion(self, n: int) -> int:
+        up, down = 1, n
+        middle = 0
+        while up < down:
+            middle = (up + down) // 2
+            if isBadVersion(middle) is True:

```

Comment:
if isBadVersion(middle) is True: -> if isBadVersion(middle):
Don't use "==" or "is" to compare boolean values with True or False (see PEP8).